scenes/Restaurant/Customer.gd

```diff
@@ -0,0 +1,79 @@
+extends Node2D
+
+# food colors for the food which gets hurled into the customer's mouth
+const FOOD_COLORS: Array = [
+	Color("a4470b"), # brown
+	Color("ff5d68"), # pink
+	Color("ffa357"), # bread
+	Color("fff6eb") # white
+]
+
+# signal emitted on the frame when the food is launched into the customer's mouth
+signal food_eaten
+
+# sounds which get played when the customer eats
+onready var _munch_sounds:Array = [
+	$Munch0,
+	$Munch1,
+	$Munch2,
+	$Munch3,
+	$Munch4
+]
+
+# the total number of seconds which have elapsed since the object was created
+var _total_seconds := 0.0
+
+# the creature's head bobs up and down slowly, these fields control how much it bobs
+var _head_bob_seconds := 6.5
+var _head_bob_pixels := 2
+
+# the index of the previously launched food color. stored to avoid showing the same color twice consecutively
+var food_color_index := 0
+
+func _ready() -> void:
+	$AnimationPlayer.play("Blink")
```

Comment:
This seems redundant with the logic in _process which will already play a default animation.

Reply:
Removed